src/vue/components/popup.vue

```diff
@@ -15,7 +15,7 @@
 
       <div class="alpheios-popup__toolbar-buttons">
           <alph-tooltip :tooltipText="l10n.getText('TOOLTIP_SHOW_DEFINITIONS')" tooltipDirection="bottom-wide"
-                        v-show="$store.getters['app/defDataReady']">
+                        v-show="$store.getters['app/fullDefDataReady']">
```

Comment:
I think we publish short definitions to the panel (according to the previous changes in `panel-large.vue`, then may be we need both checks here?

Reply:
@kirlat and I discussed this. The short defs are on the panel, but as they are also on the popup/morph panel, they are not the main point of the define panel. We want to just have them there if they are available, but not show an error if they aren't and I think the tab should only be available if there are full defs.